packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts

```diff
@@ -0,0 +1,299 @@
+import { DebugLogger } from '@affine/debug';
```

Comment:
the calculating logic looks heavy. should we put it in worker, ?

Reply:
Changed in subsequent PRs